packages/constants/prometheus.ts

```diff
@@ -0,0 +1,19 @@
+import { WindowObject } from '../types';
+import { ONE_HOUR } from './common';
+
+export const MCG_MS_PROMETHEUS_URL =
+  '/api/proxy/plugin/mcg-ms-console/mcg-managed-prom';
```

Comment:
```suggestion
  '/api/proxy/plugin/mcg-ms-console/prometheus-proxy';
```
Backend is suggesting to use this now !!